server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/routes/TasksRoutesTest.java

```diff
@@ -22,11 +22,7 @@
 import static io.restassured.RestAssured.given;
 import static io.restassured.RestAssured.when;
 import static io.restassured.RestAssured.with;
-import static org.hamcrest.Matchers.empty;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.isOneOf;
-import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.*;
```

Comment:
no wildcard

Reply:
sorry, I'm an Intellij newbie, I need to configure it.

Reply:
Finally joined the dark side of IDEs? :)

Reply:
Eclipse is broken with current James code source :/ (it does not appreciate the mix of Java and Scala)